recipes/signac/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "signac" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://bitbucket.org/glotzer/signac
```

Comment:
Normally we don't clone from version control. Would it be possible to update this to download a tarball instead. Example below.

```yaml
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://bitbucket.org/glotzer/signac/get/v{{ version }}.tar.gz
  sha256: <some checksum here>
```